### ISSUE TITLE
`_execute_browser_action` in manifest V3

### ISSUE THREAD

**Issue by regseb** [mozilla/addons-linter#4681]
### Describe the problem and steps to reproduce it:

addons-linter doesn't report any warning for the [`/commands/_execute_browser_action`](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/commands#special_shortcuts) property for a manifest V3.

```JSON
{
    "manifest_version": 3,
    "...": "...",

    "action": {
        "...": "..."
    },

    "commands": {
        "_execute_browser_action": {
            "suggested_key": {
                "default": "Alt+K"
            }
        }
    }
}
```

### What happened?

No warnings are reported.

### What did you expect to happen?

addons-linter reports a warning as for the `browser_action` property.

| Code | Message | Description | File | Line | Column |
| --- | --- | --- | --- | --- | --- |
| MANIFEST_FIELD_UNSUPPORTED | "/commands/_execute_browser_action" is in a format not supported in manifest versions > 2. | "/commands/_execute_browser_action" is in a format not supported in manifest versions > 2. | manifest.json |

### Anything else we should know?

Maybe same problem with `/commands/_execute_page_action` property.

**Comment by Rob--W** [comment]
Warning about `_execute_browser_action` makes sense, as it has been replaced with `_execute_action` in MV3.

`_execute_page_action` is still supported in Firefox's MV3 though.

**Comment by andrewbaxter** [comment]
I have a manifest v3 plugin with `_execute_browser_action` with `suggested_key` and it doesn't work in firefox.

**Comment by Rob--W** [comment]
Use `_execute_action` instead, as documented at https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/commands#special_shortcuts

**Comment by andrewbaxter** [comment]
Right, I intend to, but

> _execute_page_action is still supported in Firefox's MV3 though.

Did I misinterpret that?

**Comment by Rob--W** [comment]
> Right, I intend to, but
> 
> > _execute_page_action is still supported in Firefox's MV3 though.
> 
> Did I misinterpret that? If it isn't supported then a warning as @regseb suggested would be much appreciated.

Firefox continues to support page_action in MV3, and therefore also _execute_page_action.

**Comment by andrewbaxter** [comment]
Ah, okay, so to clarify what you wrote you need to use the corresponding `page_action` or `action` respectively with `_execute_page_action` and `_execute_action`.  If they're mixed it just doesn't work (I guess?) and the linter doesn't catch this.

It's a bit unfortunate, not that I'm a careful reader, but https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json doesn't document this and obviously chrome docs won't describe ff-specific behavior, so the linter is really the only thing we have to rely on.
